packages/dcos-integration-test/extra/api_session_fixture.py

```diff
@@ -0,0 +1,14 @@
+"""This file has only one function: to provide a correctly configured
+ClusterApi object that will be injected into the pytest 'dcos_api_session' fixture
+via the make_dcos_api_session_fixture() method
+"""
+from test_util.dcos_api_session_api import ClusterApi, get_args_from_env
+from test_util.helpers import CI_AUTH_JSON, DcosUser
+
+
+def make_session_fixture():
+    args = get_args_from_env()
+    args['web_auth_default_user'] = DcosUser(CI_AUTH_JSON)
+    dcos_api_session = ClusterApi(**args)
```

Comment:
I thought, you would change `ClusterApi` class to `DcosSessionApi` too.  Do you think the current term for the class `ClusterApi` serves well?

If you do change to `DcosSessionApi` then the instance name could be `dcos_session` can be correlated easily.  
Right now, the `dcos_api_session`  instance corresponds to a `ClusterAPI` class that demands a little familiarity with the code-base to correlate.